bin/oref0-setup.sh

```diff
@@ -531,7 +537,11 @@ elif [[ ${CGM,,} =~ "xdrip" ]]; then
     (crontab -l; crontab -l | grep -q "cd $directory && ps aux | grep -v grep | grep -q 'openaps monitor-xdrip'" || echo "* * * * * cd $directory && ps aux | grep -v grep | grep -q 'openaps monitor-xdrip' || ( date; openaps monitor-xdrip) | tee -a /var/log/openaps/xdrip-loop.log; cp -up $directory/xdrip/glucose.json $directory/monitor/glucose.json") | crontab -
     (crontab -l; crontab -l | grep -q "xDripAPS.py" || echo "@reboot python $HOME/.xDripAPS/xDripAPS.py") | crontab -
 elif ! [[ ${CGM,,} =~ "mdt" ]]; then
+  if ! [[ $ENABLE =~ dexusb ]]; then
```

Comment:
Would it be clearer if we reversed this and did "if dexusb" and the contents of your else clause, and then put the existing text in the else?